### ISSUE TITLE
Cannot connect to simulator

### ISSUE THREAD

**Issue by Something231** [Micromelon-Robotics/mm-pymodule#4]
## **Describe the bug**
You cannot connect to simulator using the py module with BLE value of 9000, which is opposite of what is expected

## **To Reproduce**
```
from micromelon import *
rc = RoverController()
rc.connectBLE(9000) 
rc.startRover() 
```

**Results in:**
```
TimeoutError: Robot not found - Scan timed out
```

## **Desktop (please complete the following information):**
 - OS: Win 11
 - Version: 1.1.1

**Comment by CaptainDeathead** [comment]
Have you tried it on linux?

**Comment by timmyhadwen** [comment]
Hey @Something231 you'll need to use ```rc.connectIP('localhost', 9000)``` instead of ```connectBLE```.

**Comment by CaptainDeathead** [comment]
@timmyhadwen why is that? I'm new to micromelons...

**Comment by Something231** [comment]
> Hey @Something231 you'll need to use ```rc.connectIP('localhost', 9000)``` instead of ```connectBLE```.

Cheers for the clarification, maybe it should be made more explicit in the sim or the pip documentation? Thanks anyway

**Comment by timmyhadwen** [comment]
TLDR is that the real rovers connect via Bluetooth, but the simulator rovers connect via TCP. Therefore we need to have different code depending on which is available!

Agreed that should be documented in the readme!
